AdoptSkin/Framework/SaveLoadHandler.cs

```diff
@@ -380,9 +380,9 @@ internal static void LoadAssets()
 
             // Add custom sprites
             foreach (string path in Directory.EnumerateFiles(Path.Combine(SHelper.DirectoryPath, "assets", "skins"), "*", SearchOption.AllDirectories))
-                PullSprite(path);
+                PullSprite(path.Replace(SHelper.DirectoryPath, "")); // Fix an absolute path error
```

Comment:
I suggest doing this instead, which will avoid issues if the path formatting doesn't match (e.g. `/` vs `\\`):
```suggestion
                PullSprite(Path.GetRelativePath(SHelper.DirectoryPath, path)); // must be a relative path
```